subprocess/subtree_oom.go

```diff
@@ -63,3 +69,61 @@ func getPidFromLog(line string) (int, bool) {
 	}
 	return pid, true
 }
+
+func ClearOOMHandler() http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		grip.Debug("clearing oom")
```

Comment:
You probably don't need to add logging statements in the agent like this since they are rarely seen